cmd/newrelic-infra/newrelic-infra.go

```diff
@@ -298,6 +298,7 @@ func initializeAgentAndRun(c *config.Config, logFwCfg config.LogForward) error {
 		FluentBitExePath:     c.FluentBitExePath,
 		FluentBitNRLibPath:   c.FluentBitNRLibPath,
 		FluentBitParsersPath: c.FluentBitParsersPath,
+		FluentBitVerbose:     c.Verbose != 0,
```

Comment:
I wasn't sure whether to enable it when `Verbose != 0` or just when `Verbose = 1`. Also, I am afraid that when `Verbose = 3`, when shipping the infra-agent logs (and those logs containing the logs being shipped, due that is in verbose mode), this might cause an infinite loop. Maybe it would be wiser to just enable it when `Verbose = 1`? Any thoughts @carlosroman @varas @cristianciutea ?

Reply:
verbose: 2 is a special "smart" verbosity mode, see https://docs.newrelic.com/docs/infrastructure/infrastructure-troubleshooting/troubleshoot-logs/infrastructure-agent-logging-behavior#smart-verbose-mode

Reply:
@jsubirat  did you experience a retro feedback loop enabling verbosity on both agent and FluentBit?

Reply:
I tested it now and actually doesn't cause an infinite loop, thanks to the fact that we're not outputting the forwarded records. However, a hacky mind could define an external fluent-bit configuration file to be included (via the `@INCLUDE` tag) with the following contents:

```
[OUTPUT]
    Name stdout
    Match *
```

And this does cause lots of stuff being written to the infra-agent log, as the records themselves are sent to the infra-agent log, which in turn are captured by fluent-bit, which in turn sends that to the log. It has an exponential growth (about 250MB/min in my VM).

I'd recommend, then, to only enable the verbose mode of Fluent Bit when running with Verbose = 2, what do you think?

Reply:
I added a change for FB verbose to be enabled as a feature trace